### PR TITLE
chore(wallet-client): add failed peg-out refund test

### PR DESCRIPTION
**Pull request by elsirion**

### CODE REVIEW

File: modules/fedimint-dummy-client/src/lib.rs [hunk 1]
```diff
@@ -300,6 +300,7 @@ impl ClientModule for DummyClientModule {
                         // Since Done also happens for inputs we will fire too often, but that's ok
```
Comment (m1sterc001guy) [inline modules/fedimint-dummy-client/src/lib.rs:300]: Nit: I don't think this comment is super applicable anymore

**Review by m1sterc001guy — APPROVED** [review]

File: fedimint-testing/src/fixtures.rs [hunk 2]
```diff
@@ -26,7 +26,7 @@ use crate::ln::real::{ClnLightningTest, LdkLightningTest, LndLightningTest};
 use crate::ln::LightningTest;
 
 /// A default timeout for things happening in tests
-pub const TIMEOUT: Duration = Duration::from_secs(10);
+pub const TIMEOUT: Duration = Duration::from_secs(30);
```
[the remark this answers is not in the document]
Comment (elsirion) [reply fedimint-testing/src/fixtures.rs:29]: I had a problem once with that timeout under high load iirc, but can remove it and we add it if needed.

File: modules/fedimint-dummy-client/src/lib.rs (hunk 1 above)
Comment (elsirion) [reply modules/fedimint-dummy-client/src/lib.rs]: Removed :+1:

File: fedimint-testing/src/fixtures.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (elsirion) [reply fedimint-testing/src/fixtures.rs]: Turns out the peg-in can just take a bit longer sometimes, I made it a new constant so I don't mask actual problems in other tests.

**Review by dpc — APPROVED** [review]

**Pull request merged**